src/commands/PublishCommand.js

```diff
@@ -595,7 +609,7 @@ export default class PublishCommand extends Command {
   }
 
   gitCommitAndTagVersionForUpdates() {
-    const tags = this.updates.map(({ package: { name } }) =>
+    const tags = this.updates.map(({ "package": { name } }) =>
```

Comment:
I can back out this change, but package is a reserved keyword and shouldn't be referenced without quotes.

Reply:
Indeed, good call!